ExampleApp/Views/BaseViewController.swift

```diff
@@ -0,0 +1,194 @@
+//
+//  BaseViewController.swift
+//  OmiseSDK
+//
+//  Created by Andrei Solovev on 4/10/23.
+//  Copyright © 2023 Omise. All rights reserved.
+//
+
+import UIKit
+import OmiseSDK
+
+enum CodePathMode: Int {
+    case storyboard = 0
+    case code = 1
+}
+
+class BaseViewController: UIViewController {
+
+    @IBOutlet private var modeChooser: UISegmentedControl!
+
+    var currentCodePathMode: CodePathMode = .code
+    var paymentAmount: Int64 = 0
+    var paymentCurrencyCode: String = ""
+    var usesCapabilityDataForPaymentMethods = false
+    var allowedPaymentMethods: [SourceTypeValue] = []
+
+    required init?(coder: NSCoder) {
+        self.usesCapabilityDataForPaymentMethods = true
+        super.init(coder: coder)
+    }
+
+    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+        self.usesCapabilityDataForPaymentMethods = true
+        super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
+    }
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Workaround of iOS 12 bug on the tint color
+        self.view.tintColor = nil
+        self.navigationController?.navigationBar.tintColor = nil
+
+        updateUIColors()
+
+        #if INTERNAL_TEST
+        modeChooser.selectedSegmentIndex = CodePathMode.code.rawValue
+        modeChooser.isEnabled = false
+        #endif
+
+        let localeCountryCode: String? = {
+            if #available(iOS 16, *) {
+                return Locale.current.region?.identifier
+            } else {
+                return Locale.current.regionCode
+            }
```

Comment:
```suggestion
            }
            return Locale.current.regionCode
```

Reply:
It's a copy-pasted Objective-C file rewritten with Swift without much refactoring tho I'd still keep it this way as it's make it more clear to read the code and identify the intention in this specific case.